### PR TITLE
#170947543 setup heroku

### PR DESCRIPTION
**Pull request by Ntare22**
**What does this PR do?**
* Setup heroku

**Description of Task to be completed?**
* add node version that heroku will be running
* add app.json file to create deploy button
* add heroku button to README.md

**How should this be manually tested?**
* Go to this branch
* Click to deploy to heroku

**Any background context you want to provide?**
* N/A

**What are the relevant pivotal tracker stories?**
* [#170947543](https://www.pivotaltracker.com/story/show/170947543)

**Screenshots (if appropriate)**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,4 @@ Barefoot Nomad - Making company travel and accomodation easy and convinient.
 Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
 
 ---
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```
Comment (Mnickii) [inline README.md]: let's remove this. We have automatic deploys so it's not necessary.

File: app.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "description": "Making company travel and accomodation easy and convinient.",
+  "repository": "https://github.com/andela/devrepublic-bn-backend",
+  "keywords": [
+    "devrepublic",
+    "andela",
+    "simulations"
+  ],
+  "addons": [],
+  "env": {}
+}
```
Comment (Mnickii) [inline app.json]: lets add a newline; you can configure your editor to automatically do this.

**Review by Mnickii — CHANGES_REQUESTED** [review]

**Review by Mnickii — DISMISSED** [review]

**Review by Baraka-Mugisha — DISMISSED** [review]

**Review by Baraka-Mugisha — DISMISSED** [review]

**Review by bbaime98 — DISMISSED** [review]

**Review by Baraka-Mugisha — DISMISSED** [review]

**Comment by Baraka-Mugisha** [comment]
Jim, Could you rebase the PR?

**Comment by Ntare22** [comment]
@Baraka-Mugisha done!

**Review by Baraka-Mugisha — DISMISSED** [review]

**Review by Baraka-Mugisha — APPROVED** [review]

**Review by Mnickii — APPROVED** [review]

**Pull request merged**